src/client/pydaos/raw/daos_cref.py

```diff
@@ -340,3 +340,6 @@ class Logfac:
 
 # Transaction handle to update for an independent transaction
 DAOS_TX_NONE = Daos_handle_t(0)
+
+# default fault domain level
+DAOS_PROP_CO_REDUN_DEFAULT = 2
```

Comment:
Isn't the default 1?

Reply:
This is for container API default, it should be still 2. for test, we used 1.